rpm/generic/zfs.spec.in

```diff
@@ -57,6 +57,7 @@
 %bcond_with    asan
 %bcond_with    systemd
 %bcond_with    pam
+%bcond_with    pyzfs
```

Comment:
The conditional you replaced with this defaulted to `%bcond_without pyzfs`, not `%bcond_with` - is there a reason for changing the default behavior here?

Reply:
There's a discrepancy between the comment and actual code. I think the author meant to enable `pyzfs` by default.
Do you see harm in enabling it by default?

Reply:
That's...what the code did.

%bcond_without defines the symbol unless you specified --without-X, %bcond_with defines it if you specified --with-X.